.vscode/launch.json

```diff
@@ -1,6 +1,13 @@
 {
 	"version": "0.1.0",
 	"configurations": [
+		{
+			"name": "Launch Chrome",
```

Comment:
Maybe the name should better clarify the purpose of this launch config.

Reply:
Oh, that was an accident! Good catch!